ext/netlify_redirector/extconf.rb

```diff
@@ -37,22 +37,12 @@ def package_config(pkg, options={})
 have_header("re2.h")
 
 # setup constant that is equal to that of the file path that holds that static libraries that will need to be compiled against
-LIB_DIRS = [LIBDIR, File.expand_path(File.join(File.dirname(__FILE__), "lib"))]
-libs = []
-case RUBY_PLATFORM
-when /darwin/
-  libs << '-lnetlify-redirects_darwin'
-when /linux/
-  libs << '-lnetlify-redirects_linux'
-else
-  raise "Unsupported platform"
-end
+LOCAL_LIB_DIR = File.expand_path(File.join(File.dirname(__FILE__), "lib"))
+LIB_DIRS = [LIBDIR, LOCAL_LIB_DIR]
+
+find_library("libnetlify-redirects", nil, LOCAL_LIB_DIR)
```

Comment:
do you need to be checking the return here? and don't you want the `LIBS_DIR` not just local?